src/components/CookieNotice/CookieNotice.tsx

```diff
@@ -0,0 +1,132 @@
+import "./CookieNotice.scss";
+import React from "react";
+import Backdrop from "@material-ui/core/Backdrop";
+import store from "store";
+import {useSelector} from "react-redux";
+import {ApplicationState} from "types/store";
+import {ActionFactory} from "store/action";
+import CookiePolicy from "./CookiePolicy";
+
+/**
+ * Converts string value to boolean value.
+ * @param input string that needs conversion to boolean value.
+ */
+const stringToBoolean = (input: string): boolean => {
+  switch (input.toLowerCase().trim()) {
+    case "true":
+    case "yes":
+    case "1":
+      return true;
+    case "false":
+    case "no":
+    case "0":
+    case null:
+      return false;
+    default:
+      return false;
+  }
+};
+
+const CookieNotice = () => {
+  /** cookie name */
+  const scrumlrCookieName = "scrumlr_cookieConsent";
```

Comment:
This should be a constant outside of the component, e.g. `const COOKIE_CONSENT_NAME='...'` on top of the file